pylab/website/tests/test_events.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+
+from django_webtest import WebTest
+
+from django.contrib.auth.models import User
+
+from pylab.core.models import Event
+
+
+class EventTests(WebTest):
+
+    def test_event_details_page(self):
+        user = User.objects.create(username='u1')
+
+        Event.objects.create(
+            author=user,
+            starts=datetime.datetime(2015, 7, 29),
+            ends=datetime.datetime(2015, 7, 30),
+            title='Test title',
+            osm_map_link='http://openstreetmap.org/',
+            description='Test description',
+        )
```

Comment:
I think, you can use `EventFactory` in my branch (not yet in master): https://github.com/python-dirbtuves/website/blob/monday-event/pylab/core/factories.py#L19